Xamarin.Forms.Core/Registrar.cs

```diff
@@ -21,6 +21,9 @@ public class Registrar<TRegistrable> where TRegistrable : class
 
 		public void Register(Type tview, Type trender)
 		{
+			//avoid caching null renderers
```

Comment:
After further revision and discussion we found that the problem is with the lookup system that needs more work, and we will go with just reverting caching null for now